### PR TITLE
Reset LS distance when not in Precland (CU-23ke6gd)

### PR DESCRIPTION
**Pull request by RafikHachana**
Reset the distance to the landing station when the `dis_ls` is not received for more than 3 seconds (done by adding a timer in QML that periodically checks the latest distance info)

### CODE REVIEW

**Comment by julia-nitsch** [comment]
Task linked: [CU-23ke6gd LS Distance not reset when not in PrecLand](https://app.clickup.com/t/23ke6gd)

**Review by julia-nitsch — COMMENTED** [review]
Minor question about the timer.

File: src/QmlControls/LandingStationIndicator.qml [hunk 1]
```diff
@@ -21,19 +21,37 @@ QGCLabel {
...
     }
     property var    currentVehicle:         QGroundControl.multiVehicleManager.activeVehicle
     
 
-    /*Item {
+    // Continuously checks if the LS Distance info is outdated
+    Item {
        Timer {
-               interval: 100; running: true; repeat: true
+               interval: 2000; running: true; repeat: true
```
Comment (julia-nitsch) [inline src/QmlControls/LandingStationIndicator.qml:51]: Sorry I don't get it from this single line. Could you please explain how increasing the timer helps in setting N/A when not being in precland mode?

**Review by julia-nitsch — COMMENTED** [review]
Minor comment regarding explanation

File: src/QmlControls/LandingStationIndicator.qml (hunk 1 above)
Comment (RafikHachana) [reply src/QmlControls/LandingStationIndicator.qml:51]: The timer code was there previously and was commented out (from the early stages of development of the LS indicator). I brought it back and increased the timeout.

Comment (julia-nitsch) [reply src/QmlControls/LandingStationIndicator.qml:51]: I see and if the code within the repeated call is not receiving an update, you set the displayed distance value to N/A?

Comment (RafikHachana) [reply src/QmlControls/LandingStationIndicator.qml:51]: In that case, I just don't display a distance (I just display LS disconnected)

**Pull request merged**